misc/host_tests.conf

```diff
@@ -9,9 +9,8 @@ add fail
 add ping
 add bacnet
 add mudgee
-
 # Additional base modules
 include subset/pentests/build.conf
-
+include subset/mac/build.conf
```

Comment:
should not be in host_tests -- those are the baseline test that are more about core system development.